_posts/2023-09-13-prospero-add-feature-pack.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title:  "Installing additional feature packs in WIldfly"
+date:   2023-09-13
+tags:   prospero galleon wildfly
+author: spyrkob
+description: Using Prospero to installing additional feature packs in existing Wildfly servers.
+---
+
+== Additional feature packs for Wildfly
+
+Wildfly server capabilities can be extended by installing additional feature packs such as wildfly-datasources-galleon-pack or wildfly-myfaces-galleon-pack.
+The feature packs can be installed using Galleon or Prospero.
+
+In this article we will use Prospero to install a MariaDB datasource provided by wildfly-datasources-galleon-pack.
+
+== Prerequisites
+
+1. Download and unzip Wildfly 29.0.1.Final from https://www.wildfly.org/downloads/
+1. Checkout Prospero from https://github.com/wildfly-extras/prospero
```

Comment:
wildfly.org now supports copy/paste to clipboard. For instructions that users have to run, that's now better to provide the actual instructions with something like:

```
[source]
----
git clone https://github.com/wildfly-extras/prospero
cd prospero
mvn install -Pdist
----
```